pkg/tlscheck/tlscheck.go

```diff
@@ -18,9 +19,21 @@ const (
 	timeout = 2 * time.Second
 )
 
+// addrValid validates the URL. It assumes scheme prefixes have been removed
```

Comment:
[nit] I think it doesn't assume the prefixes are removed at this point 🙂

Reply:
I changed the validation to be a bit stricter in [f472ed4](https://github.com/stackrox/stackrox/pull/11668/commits/f472ed43bd7e89a45cdb62fb8c78bf1aa1559d6d)